patches/storage.patch

```diff
@@ -5,7 +5,7 @@ index 6df6b5c4a7..54b5a9699b 100644
 @@ -1,3 +1 @@
```

Comment:
Do we actually use/need these patches anymore?

Reply:
I'm not sure, this file hadn't been touched in over 2 years, so probably not?

Reply:
Yeah, let's delete them. I think they used to document known divergence from IDPE, but they are not up to date AFAIK